microsoft-365/compliance/tls-1-2-in-office-365-gcc.md

```diff
@@ -1,28 +1,26 @@
 ---
 title: Deprecating TLS 1.0 and 1.1 in Office 365 GCC High and DoD
 description: Discusses how Microsoft is moving the date forward to discontinue support for TLS 1.1 and 1.0 in GCC High and DoD environments in Office 365 and preparing to use TLS 1.2.
-author: simonxjx
-manager: dcscontentpm
+author: workshay
+manager: laurawi
 localization_priority: Normal
 search.appverid: 
 - MET150
 audience: ITPro
-ms.service: O365-seccomp
+ms.collection: M365-security-compliance
+ms.service: information-protection
 ms.topic: article
-ms.author: v-six
-ms.reviewer: lobrion
+ms.reviewer: krowley
+ms.auther: shmehta
```

Comment:
Correction. Typo in metadata. @WorkShay change ms.auther: shmehta to

ms.author: shmehta